Yoast/YoastSEO.js
# Add full text specs for YoastSEO.js

## Summary

This PR can be summarized in the following changelog entry:

* no user-facing changes

## Relevant technical choices:

* Adds a pipeline for full-text analysis within specs in YoastSEO.js. This will help to keep track of changes that intentionally or unintentionally alter user-facing functionality, using real-world texts (different from the current specs). 
* The pipeline calls every assessment and every researcher that currently generate user-facing results of readability and SEO analysis.
* This PR also includes a text (in English) to serve ~~inspiration~~ example for all developers who work in YoastSEO.js to add full-text tests when developing new functionality or fixing bugs.

## Test instructions

This PR can be tested by following these steps:

* Add another text to the full-text specs (don't forget to add it to the list of texts to run).
* Make sure that all assessments are actually called (e.g., do `yarn test spec/fullTextTests/runFullTextTests.js` and that they return expected user-facing feedback.
* Do not forget to push your new text!

Fixes #1604
